src/FontMetrics.php

```diff
@@ -113,7 +113,7 @@ public function load_font_families()
      */
     public function loadFontFamilies()
     {
-        $file = $this->options->getRootDir() . "/lib/fonts/installed-fonts.dist.json";
+        $file = $this->options->getFontDir() . "/installed-fonts.dist.json";
```

Comment:
I'd have to think on this one more. The purpose of this particular line of code was to pull the distributed JSON to ensure Dompdf always has a baseline of supported fonts. The generated JSON file can override that, but there's no guarantee that it will include the ".dist" file.